src/v/security/audit/audit_log_manager.cc

```diff
@@ -847,6 +889,47 @@ bool audit_log_manager::report_redpanda_app_event(is_started app_started) {
         : application_lifecycle::activity_id::stop);
 }
 
+model::partition_id audit_log_manager::compute_partition_id() {
+    static thread_local model::partition_id _next_pid{0};
+
+    model::topic_namespace_view ns_tp{model::kafka_audit_logging_nt};
+    auto cfg = _metadata_cache->local().get_topic_cfg(ns_tp);
+    if (cfg.has_value()) {
+        vlog(
+          adtlog.debug,
+          "{} missing from metadata cache, fall back to partition 0",
+          ns_tp);
+        return _next_pid;
```

Comment:
nit: Log message says "partition 0" but you're returning `_next_pid` which may not be `0` correct?  Either update the log message or return 0?

Reply:
log left over from PoC before I implemented the RR. good catch, thanks.

Reply:
fixed